BaseFilter/sections/css_extended.txt

```diff
@@ -2424,8 +2424,8 @@ yeptube.com#?#.wrapper > div.container:has(> div.c-title > div.c-medtitle-output
 ! facebook.com
 ! https://github.com/AdguardTeam/AdguardFilters/issues/135446#issuecomment-1332051619
 ! Facebook Marketplace
-[$path=/marketplace/]facebook.com#$?#div[class][style^="max-width:"]:has(> span > div  a[href^="/ads/about/"]) { display: none !important; }
-[$path=/marketplace/]facebook.com#$?#div[class][style^="max-width:"]:has(> span > div > a[target="_blank"]:not([href^="/marketplace/"])) { display: none !important; }
+[$path=/marketplace/]facebook.com,facebookwkhpilnemxj7asaniu7vnjjbiltxjqhye3mhbshg7kx5tfyd.onion#$?#div[class][style^="max-width:"]:has(> span > div  a[href^="/ads/about/"]) { display: none !important; }
```

Comment:
These better write as new line if nothing has been changed to use:

`[$path=/marketplace/]facebook.com,[$path=/marketplace/]facebookwkhpilnemxj7asaniu7vnjjbiltxjqhye3mhbshg7kx5tfyd.onion`

Reply:
You sound like you know more about `$path` than I do. Give me 20min.

Reply:
Recently, SKEIDs combined identical rules and somewhere with the help of path was narrowed down and had to ditch it, however.

> https://github.com/AdguardTeam/AdguardFilters/pull/137586#discussion_r1051386265

So it would be useful to document how the path affects decimal domains.

It may also be about backward compatibility so that the simpler filter does not deactivate in a non-updated application/add-on.